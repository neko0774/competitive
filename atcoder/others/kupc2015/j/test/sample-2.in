8 5
18 29 52 64 4 59 89 15
1 2
1 3
1 4
2 5
3 6
3 7
4 8
99 6 7
102 5 8
15 2 6
84 7 8
19 1 1
