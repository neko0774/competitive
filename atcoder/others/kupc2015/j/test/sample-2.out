40
2
15
14
1
